test/parallel/test-path-parse-format.js

```diff
@@ -5,11 +5,12 @@ var path = require('path');
 
 var winPaths = [
```

Comment:
Since we're in here making changes anyway, the require decls and these top level decls could be change to `const` now